test-runner-poc/environments.js

```diff
@@ -75,6 +71,15 @@ class ReplicaSetEnvironment extends EnvironmentBase {
     }
   }
 }
+
+function generateNodesArray(hosts, configFunc){
+  let array = [];
```

Comment:
let's not use `array` as a variable name.

Reply:
looks like this comment was accidentally resolved

Reply:
changed the name to `nodesArray`